driver-core/src/main/com/mongodb/internal/operation/CreateIndexesOperation.java

```diff
@@ -284,6 +309,9 @@ private BsonDocument getCommand(final ConnectionDescription description) {
         command.put("indexes", new BsonArray(values));
         putIfNotZero(command, "maxTimeMS", maxTimeMS);
         appendWriteConcernToCommand(writeConcern, command, description);
+        if (commitQuorum != null && serverIsAtLeastVersionFourDotFour(description)) {
```

Comment:
The specification requires that the driver throw an exception in this case instead of silently ignoring it.

Reply:
Make sure you add a test for it as well.